infra/modules/database/role_manager/role_manager.py

```diff
@@ -0,0 +1,134 @@
+import itertools
```

Comment:
General question, is there or is there going to be additional functionality added for locking down the `public` schema by default?

Reply:
Not sure, created a ticket for it for now, we could discuss there: https://github.com/navapbc/template-infra/issues/300

Reply:
I think I answered my own question after deploying. If I am understanding, this: `postgres=UC/postgres, =UC/postgres`, is what locks down public.

Reply:
Cool, I threw a comment into [#300 ](https://github.com/navapbc/template-infra/issues/300). I also did a little more digging and realized this `postgres=UC/postgres, =UC/postgres` is the standard permissions and not a result from something we did.